packages/ai/src/chat/google-vertex/nova/prompt/route.ts

```
@@ -601,3 +594,125 @@
+    .single();
+
+  if (roleError && roleError.code !== 'PGRST116') {
+    console.error('Database Error when checking role:', roleError);
+    return {
+      canSubmit: false,
+      remainingAttempts: 0,
+      message: 'Error checking user permissions',
+    };
+  }
+
+  const isAdmin = roleData && roleData.allow_challenge_management;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)